libs/evcs-service/src/test/java/uk/gov/di/ipv/core/library/pact/ContractTest.java

```diff
@@ -0,0 +1,463 @@
+                                EVCS_API_KEY,
+                                "Authorization",
+                                "Bearer " + "invalid-access-token"))
+                .willRespondWith()
+                .status(403)
+                .toPact();
+    }
+
+    @Test
+    @PactTestFor(pactMethod = "invalidAuthorizationTokenReturns403")
+    void testRetrieveVcRequestReturns403(MockServer mockServer) {
+        // Mock Data
+        lenient()
+                .when(mockConfigService.getSecret(ConfigurationVariable.EVCS_API_KEY))
+                .thenReturn(EVCS_API_KEY);
+
+        // Under Test
+        EvcsClient evcsClient = new EvcsClient(mockConfigService);
+        assertThrows(
+                EvcsServiceException.class,
+                () -> {
+                    evcsClient.getUserVcs(
+                            INVALID_USER_ID, "invalid-access-token", VC_STATES_FOR_QUERY);
+                });
+    }
+
+    @Pact(provider = "EvcsProvider", consumer = "IpvCoreBack")
+    public RequestResponsePact validCreateUserVcReturnsMessageIdWith200(
+            PactDslWithProvider builder) {
+        return builder.given("test-evcs-api-key is a valid API key")
+                .uponReceiving("A request to create EVCS user VCs")
+                .path("/vcs/" + TEST_USER_ID)
+                .method("POST")
+                .headers(
+                        Map.of(
+                                "x-api-key",
+                                EVCS_API_KEY,
+                                CONTENT_TYPE,
+                                ContentType.APPLICATION_JSON.toString()))
+                .body(getRequestBodyForUserVC())
+                .willRespondWith()
+                .status(200)
+                .body(getResponseBodyCreateUserVC())
+                .toPact();
+    }
+
+    private DslPart getRequestBodyForUserVC() {
+        return newJsonArray(
+                        array -> {
+                            array.object(
+                                    vcObject -> {
+                                        vcObject.stringType(
+                                                "vc", VcFixtures.DCMAW_PASSPORT_VC.getVcString());
+                                        vcObject.stringType("state", "CURRENT");
+                                        vcObject.object(
+                                                "metadata",
+                                                metadata -> {
+                                                    metadata.stringType("reason", "testing");
+                                                    metadata.stringType(
+                                                            "timestampMs", "1711721297123");
+                                                    metadata.stringType(
+                                                            "txmaEventId",
+                                                            "1a116fe7-2ff9-4f7c-940d-d55fa7d03d66");
```

Comment:
I think it's more reasonable for us to provide some metadata here - but I'd still like it to be clear that this is arbitrary test data.